book/text.md

```diff
@@ -859,12 +896,12 @@ should look something like this:
 Exercises
 =========
 
-*Centered Text:* This book's page titles are centered: find them
-between `<h1 class="title">` and `</h1>`. Make your browser center the
-text in these titles. Each line has to be centered individually,
-because different lines will have different lengths.
+*Centered Text:* This book's page titles are centered; make your
+browser do the same for text between `<h1 class="title">` and `</h1>`.
```

Comment:
Add a footnote about the `<center>` tag's interesting origins in the days before CSS?

Reply:
I added a footnote about how there was such a tag, was there something more?